lib/main_ltp.pm

```diff
@@ -0,0 +1,144 @@
+# Copyright © 2017 SUSE LLC
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, see <http://www.gnu.org/licenses/>.
+#
+# See kernel/install_ltp.pm and kernel/run_ltp.pm for documentation.
+package main_ltp;
+use base 'Exporter';
+use Exporter;
+use testapi qw(get_var);
+use autotest;
+use utils;
+use LTP::TestInfo qw(testinfo);
+use File::Basename 'basename';
+use 5.018;
+
+our @EXPORT = qw(maybe_load_kernel_tests);
+
+sub loadtest {
+    my ($test, %args) = @_;
+    autotest::loadtest("tests/kernel/$test.pm", %args);
```

Comment:
why not just extend existing loadtest function which we have in main_common ?

Reply:
I just didn't want to to a circular `use`, so we would need to create a third file that is imported by main_ltp.pm and main_common.pm.